db/schema.rb

```diff
@@ -21,6 +21,17 @@
   enable_extension "postgis"
   enable_extension "uuid-ossp"
 
+  create_table "account_configs", force: :cascade do |t|
+    t.string "account_id", null: false
+    t.text "description", null: false
+    t.string "scope", null: false, array: true
```

Comment:
Can you make the `scopes` a `t.text` instead? My recommendation will be that these are URLs, and it's safer if they're longer than the `t.string` field allows